code/game/turfs/open/lava.dm

```diff
@@ -175,3 +175,69 @@
 
 /turf/open/lava/smooth/airless
 	initial_gas_mix = AIRLESS_ATMOS
+
+/turf/open/lava/smooth/jungle
+	initial_gas_mix = LAVALAND_DEFAULT_ATMOS
+	name = "boiling mud"
+	desc = "A pool of bubbling slurry with steam coming off of it. Looks dangerous!"
+	planetary_atmos = TRUE
+	baseturfs = /turf/open/lava/smooth/jungle
+	icon = 'icons/turf/floors/hotmud.dmi'
+
+/turf/open/lava/smooth/jungle/burn_stuff(AM)
```

Comment:
This looks like copypaste from lava, what's the actual result you want to achieve here ?

Reply:
If you're trying to remove some part of lava code either add control variable to base type and set it on the mud
```
/turf/open/lava
   var/burn_clothes = TRUE

/turf/open/lava/proc/burn_stuff()
   ...
   if(burn_clothes)
     qdel(shoes)
   ...

/turf/open/lava/mud
  burn_clothes = FALSE
```
Or add a function (burn_clothes() in example) that will be overridden on your type that does the same.